Utils/build-release-macos.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+set -eux
+
+cd "$(dirname $0)/.."
+
+cmake -B Build/release-arm64 -DCMAKE_BUILD_TYPE=Release -DWITH_FULL_RELEASE=On -DCMAKE_OSX_ARCHITECTURES=arm64
+cmake --build Build/release-arm64 -j $(sysctl -n hw.logicalcpu)
+cmake --build Build/release-arm64 --target package
```

Comment:
Do we need to build both variants? It might better to just detect the version we need to build for the current system and just use that

Reply:
OH OH these are release scripts for the repo, and not like quickinstall scripts for users. That makes sense now. SGTM.